src/main/java/mate/academy/bookstore/BookstoreApplication.java

```diff
@@ -0,0 +1,33 @@
+package mate.academy.bookstore;
+
+import java.math.BigDecimal;
+import mate.academy.bookstore.model.Book;
+import mate.academy.bookstore.service.BookService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+
+@SpringBootApplication
+public class BookstoreApplication {
+    @Autowired
+    private BookService bookService;
```

Comment:
Don't forget about dependency injection - provide this service through constructor and add @ Autowired above constructor.